gitpod-io/gitpod
# [dashboard] always render context URL without prefixes

## Description


Right now:
- some URLs are rendered with prefixes on UI
- there is code duplicating prefix parsing of context URIs used on UI

This PR assumes removes duplicated code and always render context URLs without prefixes.

It is a breaking change, but I am not sure to which extend, i.e. are there really workspace without normlalized context URLs in DB? @geropl ?

Alternatively I can open another PR which uses normalized context URL or fallbacks to parsing but only for old prefix handlers for new we won't do it anymore.

## Related Issue(s)


For JetBrains we will use a special prefix handler so all workspaces started from JB Gateway will be started with it. This prefix leaks to UI here and there.

## How to test


- Start a workspace with prefix.
- Make sure that you don't see it anywhere on UI.

## Release Notes

```release-note
NONE
```

## Documentation
